src/pages/api/reset_password.js

```diff
@@ -37,6 +38,7 @@ export default async function reset_password(req, res) {
   }
 
   console.log("[reset_password]", email);
+  // TODO: send email
```

Comment:
no need to send mail manually,  it's s handle by hasura. Mail il send if new user is inserted of if secret_token is updated

Reply:
indeed !